src/screens/Profile/PaymentOptions/index.tsx

```diff
@@ -0,0 +1,205 @@
+            />
+          ))}
+        {isLoading && (
+          <ActivityIndicator
+            key={'spinner'}
+            color={theme.text.colors.secondary}
+            size="large"
+            style={{paddingTop: theme.spacings.xLarge}}
+          />
+        )}
+      </ScrollView>
+    </View>
+  );
+}
+
+const Card = (props: {
+  card: RecurringPayment;
+  removePaymentHandler: (card: RecurringPayment) => void;
+}) => {
+  const {card, removePaymentHandler} = props;
+  const paymentName = getPaymentTypeName(card.payment_type);
+  const style = useStyle();
+  const {theme} = useTheme();
+  const {t} = useTranslation();
+  const fontScale = useFontScale();
+
+  return (
+    <View style={style.card}>
+      <View style={style.cardTop}>
+        <View>
+          <ThemeText
+            accessibilityLabel={t(
+              PaymentOptionsTexts.a11y.cardInfo(paymentName, card.masked_pan),
+            )}
+          >
+            {paymentName} **** {card.masked_pan}
+          </ThemeText>
+        </View>
+
+        <View style={style.cardIcons}>
+          <PaymentBrand icon={card.payment_type} />
+          <TouchableOpacity
+            accessibilityLabel={t(
+              PaymentOptionsTexts.a11y.deleteCardIcon(
+                paymentName,
+                card.masked_pan,
+              ),
+            )}
+            style={{marginLeft: theme.spacings.medium}}
+            onPress={() => {
+              destructiveAlert({
+                alertTitleString: t(PaymentOptionsTexts.deleteModal.title),
+                alertMessageString: t(PaymentOptionsTexts.deleteModal.message),
+                cancelAlertString: t(
+                  PaymentOptionsTexts.deleteModal.cancelButton,
+                ),
+                confirmAlertString: t(
+                  PaymentOptionsTexts.deleteModal.confirmButton,
+                ),
+                destructiveArrowFunction: () => removePaymentHandler(card),
+              });
+            }}
+          >
+            <SvgDelete
+              height={21 * fontScale}
+              width={21 * fontScale}
+              fill={
+                theme.interactive.interactive_destructive.default.background
+              }
+            />
+          </TouchableOpacity>
+        </View>
+      </View>
+
+      <ThemeText color="secondary" type="body__secondary">
+        {getExpireDate(card.expires_at)}
+      </ThemeText>
+    </View>
+  );
+};
+
+const useStyle = StyleSheet.createThemeHook((theme: Theme) => ({
+  container: {
+    backgroundColor: theme.static.background.background_1.background,
+    flex: 1,
+  },
+  contentMargin: {
+    margin: theme.spacings.medium,
+  },
+  card: {
+    marginVertical: theme.spacings.xSmall,
+    borderRadius: theme.border.radius.regular,
+    backgroundColor: theme.static.background.background_0.background,
+    paddingHorizontal: theme.spacings.medium,
+    paddingTop: theme.spacings.medium,
+    paddingBottom: theme.spacings.medium,
```

Comment:
can be shorthand to **padding : theme.spacings.medium**. Doesn't do a lot but less lines of code :)